tools/utils/daymet_to_ats_box.py

```diff
@@ -173,12 +175,13 @@ def getAttrs(bounds, start, end):
     attrs['DayMet end date'] = str(end)
     return attrs    
 
-def writeATS(time, dat, attrs, filename):
+def writeATS(time, dat, x, y, attrs, filename):
     """Accepts a dictionary of ATS data and writes it to HDF5 file."""
     logging.info('Writing ATS file: {}'.format(filename))
     with h5py.File(filename, 'w') as fid:
         fid.create_dataset('time [s]', data=time)
-        
+        fid.create_dataset('row coordinate', data=x)        
+        fid.create_dataset('col coordinate', data=y)
```

Comment:
Can you confirm the shape of these arrays?  Amanzi is going to expect that these arrays are 1D, of shape (nx,) and (ny,) respectively, where dat(:,i,j) is at x(i), y(j).  It may be the case that DayMet writes out 2D objects, so that x and y are of shape (nx,ny).  I guess I'm just asking that you do something like (at line 180):

    assert(len(x.shape) == 1)
    assert(len(y.shape) == 1)
    

Then,  at line 186,

    assert(dat[key].shape[0] == time.shape[0])
    assert(dat[key].shape[1] == x.shape[0])
    assert(dat[key].shape[2] == y.shape[0])

This would just confirm that all the data are shaped correctly before writing.